docs/man/rpmdeps.1.scd

```diff
@@ -1,50 +1,76 @@
 RPMDEPS(1)
 
 # NAME
-
 rpmdeps - Generate RPM Package Dependencies
 
 # SYNOPSIS
+*rpmdeps* [options]
 
-*rpmdeps* [options] {*-P*|*--provides*} {*-R*|*--requires*} [_FILE_ ...]
+*rpmdeps* [options] _FILE_ ...
 
 # DESCRIPTION
+*rpmdeps* generates package dependencies for one or more files, received
+either as a list of filenames on the standard input, or command line
+arguments.
+
+*rpmdeps* is mainly useful for troubleshooting dependency generation
+and developing new dependency generators.
+
+Note that it is not actually used by *rpmbuild*(1) for automatic
+dependency generation except in a legacy compatibility mode.
 
-*rpmdeps* generates package dependencies for the set of _FILE_
-arguments. Each _FILE_ argument is searched for Elf32/Elf64, script
-interpreter, or per-script dependencies, and the dependencies are
-printed to stdout.
+# ARGUMENTS
+_FILE_
+	Any regular file.
 
 # OPTIONS
 See *rpm-common*(8) for the options common to all operations.
 
-*-P, --provides*
-	Print the provides
+*--alldeps*
+	Print all the dependencies.
 
-*-R, --requires*
-	Print the requires
+*--conflicts*
+	Print the conflicts dependencies.
+
+*--enhances*
+	Print the enhances dependencies.
+
+*--obsoletes*
+	Print the obsoletes dependencies.
+
+*--orderwithrequires*
+	Print the orderwithrequires dependencies.
+
+*-P, --provides*
+	Print the provides dependencies.
 
 *--recommends*
-	Print the recommends
+	Print the recommends dependencies.
+
+*-R, --requires*
+	Print the requires dependencies.
 
 *--suggests*
-	Print the suggests
+	Print the suggests dependencies.
 
 *--supplements*
-	Print the supplements
+	Print the supplements dependencies.
 
-*--enhances*
-	Print the enhances
+# ENVIRONMENT
+*RPM_BUILD_ROOT*
+	The prefix path to strip out when generating dependencies
+	based on actual paths.
 
-*--conflicts*
-	Print the conflicts
+# EXIT STATUS
+On success, 0 is returned, a non-zero failure code otherwise.
 
-*--obsoletes*
-	Print the obsoletes
+# EXAMPLES
+*rpmdeps --requires /bin/ls*
```

Comment:
This utility isn't normally installed into the `PATH` so I wonder if we should use `/usr/lib/rpm/rpmdeps` here instead? OTOH, where it's installed depends on rpm's packaging, so I guess this is generic enough...